packages/synapse-react-client/src/components/FeaturedDataTabs/SingleQueryFacetPlotsCards.tsx

```diff
@@ -20,7 +20,7 @@ export type SingleQueryFacetPlotsCardsProps = {
   sql?: string
 } & Pick<QueryVisualizationContextType, 'unitDescription'>
 
-export function getQueryRequest(sql: string): QueryBundleRequest {
```

Comment:
removed `export`, unused outside of this module. Only exporting the component allows Vite HMR to work properly